src/update.ts

```diff
@@ -345,7 +345,7 @@ export const makeUpdate =
             }
           : never,
       ): UpdateQuery<T, number> {
-        const keys = Object.keys(values);
+        const keys = Object.keys(values).filter((key) => (values as any)[key] !== undefined);
```

Comment:
nit: We can avoid a 2nd iteration and 2nd `value` lookup by filtering out undefined values while we create the tokens array.

```typescript
const valuesTokens = [];

for (const key of Object.keys(values)) {
    const value = (values as any)[key];
    if (value === undefined) continue;

    const column = (table as any)[key] as Column<any, any, any, any, any, any>;
    
    valuesTokens.push(new CollectionToken([
        new StringToken(wrapQuotes(column.getSnakeCaseName())),
        new StringToken(`=`),
        ...(isTokenable(value) ? value.toTokens() : [new ParameterToken(value)]),
    ]);
}

return new UpdateQuery(queryExecutor, [], table, 'AFFECTED_COUNT', [
    new StringToken(`UPDATE`),
    ...table.toTokens(),
    new StringToken(`SET`),
    new SeparatorToken(`,`, valuesTokens);
]);
```